monitor/index.js

```diff
@@ -5,6 +5,8 @@ const { isV1Bridge } = require('./utils/serverUtils')
 
 const app = express()
 
+const MONITOR_VALIDATOR_HOME_TX_LIMIT = Number(process.env.MONITOR_VALIDATOR_HOME_TX_LIMIT) || 0
```

Comment:
I think we should not request the gas price and fill `*VBalances` in `validators.js` if the corresponding `MONITOR_VALIDATOR_*_TX_LIMIT` is configured as zero. We can reduce network traffic if these requests are not sent.

Reply:
Removed the requests related to tx costs if not needed. Only `balance` attribute will be displayed if `MONITOR_VALIDATOR_*_TX_LIMIT` is configured as zero. https://github.com/poanetwork/tokenbridge/pull/259/commits/441c944c4bc221b3550197d4eb3afb67e3692eae